chain/westend-dev/defaults.go

```diff
@@ -25,6 +25,10 @@ func DefaultConfig() *cfg.Config {
 	config.ID = defaultID
 	config.Name = defaultName
 	config.ChainSpec = defaultChainSpec
+	config.RPC.RPCExternal = true
+	config.RPC.UnsafeRPC = true
```

Comment:
Do we want unsafe RPCs set to true by default?

Reply:
It's for the test networks. It just makes it easy to test something by not having to lookup flags